Sources/VLCCloudStorageTableViewController.m

```diff
@@ -91,14 +87,18 @@ - (void)viewDidLoad
 
 - (void)updateForTheme
 {
+    _sortBarButtonItem.tintColor = PresentationTheme.current.colors.orangeUI;
+    _numberOfFilesBarButtonItem.tintColor = PresentationTheme.current.colors.orangeUI;
     self.tableView.separatorColor = PresentationTheme.current.colors.background;
     self.tableView.backgroundColor = PresentationTheme.current.colors.background;
     self.view.backgroundColor = PresentationTheme.current.colors.background;
     _refreshControl.backgroundColor = PresentationTheme.current.colors.background;
     _activityIndicator.activityIndicatorViewStyle = PresentationTheme.current == PresentationTheme.brightTheme ? UIActivityIndicatorViewStyleGray : UIActivityIndicatorViewStyleWhiteLarge;
     self.loginToCloudStorageView.backgroundColor = PresentationTheme.current.colors.background;
     self.navigationController.toolbar.barStyle = PresentationTheme.current.colors.toolBarStyle;
+    _progressView.tintColor = PresentationTheme.current.colors.orangeUI;
     _progressView.progressLabel.textColor = PresentationTheme.current.colors.cellTextColor;
+    [self.tableView reloadData];
```

Comment:
added `reloadData` as cells do not listen to theme update. screenshot with error was attached to ticket.